locale/fr/foreman_tasks.po

```diff
@@ -7,28 +7,29 @@
 # Claer <transiblu@claer.hammock.fr>, 2016
 msgid ""
 msgstr ""
-"Project-Id-Version: foreman_tasks 0.3.4\n"
+"Project-Id-Version: foreman_tasks 7.1.0\n"
 "Report-Msgid-Bugs-To: \n"
-"PO-Revision-Date: 2020-06-22 15:15+0000\n"
-"Last-Translator: Amit Upadhye <aupadhye@redhat.com>\n"
+"PO-Revision-Date: 2016-02-12 14:04+0000\n"
+"Last-Translator: Claer <transiblu@claer.hammock.fr>, 2016\n"
 "Language-Team: French (http://www.transifex.com/foreman/foreman/language/fr/)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: fr\n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % 100000"
+"0 == 0 ? 1 : 2;\n"
 
 msgid "${taskReload ? 'Stop' : 'Start'}  auto-reloading"
-msgstr "${taskReload ? 'Stop' : 'Start'} auto-rechargement"
+msgstr "${taskReload ? 'Stop' : 'Start'}  auto-reloading"
```

Comment:
This looks like bad string interpolation from here:
https://github.com/ekohl/foreman-tasks/blob/87f1e943d2675facc8bbdb5c7b0cba9ca28a3e4c/webpack/ForemanTasks/Components/TaskDetails/Components/TaskButtons.js#L51

It's been there for a long time (> 2 years) so consider this a bug report.

Reply:
I ended up opening https://github.com/theforeman/foreman-tasks/pull/700 while waiting for translations.